packages/broker/src/ConfigWizard.ts

```diff
@@ -82,200 +87,168 @@ const DefaultConfig: Config = {
+        try {
+            const filePath = input || answers.selectDestinationPath
+            const parentDirPath = path.dirname(filePath)
 
-        return selectedPlugins
+            answers.parentDirPath = parentDirPath
+            answers.parentDirExists = existsSync(parentDirPath)
+            answers.fileExists = existsSync(filePath)
 
+            return (answers.parentDirExists || answers.fileExists)
+        } catch (e) {
+            return e.message
+        }
     }
+}
 
-    async storeConfig(): Promise<string>{
-        const destinationFolder: string = await this.selectDestinationFolder()
-        // ensure the destination folder exists
-        if (!existsSync(destinationFolder)){
-            throw new Error(`Destination folder [${destinationFolder}] does not exist`)
-        }
-        const filename = `broker-config.json`
-        const finalPath = path.join(destinationFolder, filename)
-        // ask the user if they want to overwrite when the file already exists
-        if (existsSync(finalPath)){
-            const overwrite = await this.inquirerSinglePrompt({
-                type: 'confirm',
-                name: 'overwrite',
-                message: `Config file ${finalPath} already exists. Overwrite?`,
-                default: false
-            })
+const selectValidDestinationPath = async (): Promise<inquirer.Answers> => {
+    const answers = await inquirer.prompt([selectDestinationPathPrompt])
 
-            if (!overwrite) {
-                return this.storeConfig()
+    if (answers.fileExists) {
+        const overwriteAnswers = await inquirer.prompt([
+            {
+                type: 'confirm',
+                name: 'confirmOverwrite',
+                message: `The selected destination ${answers.selectDestinationPath} already exists, do you want to overwrite it?`,
+                default: false,
             }
-        }
+        ])
 
-        writeFileSync(finalPath, JSON.stringify(this.config, null, 2))
-        return finalPath
+        if (!overwriteAnswers.confirmOverwrite) {
+            return selectValidDestinationPath()
+        }
     }
 
-    async selectDestinationFolder(): Promise<string> {
-        const defaultDestinationFolder = path.join(os.homedir(), '.streamr')
-        if (!existsSync(defaultDestinationFolder)){
-            mkdirSync(defaultDestinationFolder)
-        }
+    return answers
+}
 
-        return this.promptExistingPathWithDefault({
-            type: 'input',
-            name: 'destinationFolder',
-            message: `Select a path to store the generated config in [Enter for default: ${defaultDestinationFolder}]`,
-        }, defaultDestinationFolder)
+export const createStorageFile = (config: Config, answers: inquirer.Answers): string => {
+    if (!answers.parentDirExists) {
+        mkdirSync(answers.parentDirPath)
     }
+   
+    writeFileSync(answers.selectDestinationPath, JSON.stringify(config))
```

Comment:
Should write the JSON with indentations (`JSON.stringify(this.config, null, 2)`), like it was previously